WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/quickstart/QuickStartRepository.kt

```diff
@@ -92,7 +92,8 @@ class QuickStartRepository
     val onQuickStartTabStep = _onQuickStartTabStep as LiveData<QuickStartTabStep?>
     val activeTask = _activeTask as LiveData<QuickStartTask?>
     val isQuickStartNoticeShown = _isQuickStartNoticeShown
-    var quickStartTaskOrigin = if (isMySiteTabsEnabled) MySiteTabType.DASHBOARD else MySiteTabType.ALL
+    var currentTab = if (isMySiteTabsEnabled) DASHBOARD else MySiteTabType.ALL
```

Comment:
Nitpick: Can we use the full qualifier for `MySiteTabType.DASHBOARD` here and below + for the places where using `SITE_MENU`. 🙏

Reply:
@zwarm I have added the full qualifier in the commit [b946654](https://github.com/wordpress-mobile/WordPress-Android/pull/16462/commits/b946654ea6a2501dd95eadf8c443229633fdde84).

I intentionally kept `SITE_MENU` instead of `MySiteTabType.SITE_MENU` because there were many usages of the enum `MySiteTabType.SITE_MENU` and `MySiteTabType.DASHBOARD`, So I felt that this should make it more readable. 

I would like to know your thoughts on when or if we should import the ENUM constants or always use the full qualifier?.

Reply:
> I would like to know your thoughts on when or if we should import the ENUM constants or always use the full qualifier?.

My thoughts run like this: if we are using the full qualifier then we should use it in every case; otherwise reading it I wonder "Is this a different ENUM constant or does it refer to the same as previous". Which is why it was a nitpick. :)

Thanks.